engine/src/main/java/io/zeebe/engine/state/ZeebeState.java

```diff
@@ -131,6 +131,16 @@ public KeyGenerator getKeyGenerator() {
     return keyGenerator;
   }
 
+  // TODO (saig0): restrict access
+  public void restoreKeyGenerator(final long key) {
+    keyGenerator.setKey(key);
+  }
+
+  // TODO (saig0): restrict access
+  public long peekKeyGenerator() {
+    return keyGenerator.getCurrentKey();
+  }
+
```

Comment:
It seems these method are not used, perhaps leftovers from development.

Reply:
Already gone.